RF24.cpp

```diff
@@ -607,7 +607,8 @@ void RF24::_init_obj()
     _spi = &SPI;
 #endif // defined (RF24_SPI_PTR)
 
-    pipe0_reading_address[0] = 0;
+    memset(pipe0_reading_address, 0, 5);
+    memset(txAddress, 0xE7, 5);
```

Comment:
This might also be causing a slight increase in compile size (flash consumption). It is only done to properly initialize the address caches.

Reply:
moved this to the declarations in RF24.h. see https://github.com/nRF24/RF24/pull/1030/commits/76dfead2b5aaa5ef76999a3562ad4f4919c19ada

Reply:
Initializing the cached addresses in declaration actually increased the flash size. Instead I'm just removing the initial cached values; users should/would end up setting them anyway.